contentcuration/contentcuration/tests/viewsets/test_assessmentitem.py

```diff
@@ -0,0 +1,332 @@
+from __future__ import absolute_import
```

Comment:
What's the reason you wrote tests now? Was there a specific bug, or just to cover our bases?

Reply:
As I was doing some refactor of the code, I wanted to guarantee the current behaviour before I started messing with it. It also uncovered some errors too!

Reply:
Makes sense!